server/routes/producers-router.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken')
```

Comment:
Be careful not to include dependencies you're not using. I recommend configuring linters for your project right away.